fitbenchmarking/fitbenchmarking
# Added check for `nfev` and `nit` attributes in scipy controller

## Description of work

Fixes #1400 




## Testing Instructions



1. Run `fitbenchmarking` and verify that the COBYLA minimizer no longer raises an exception for every problem
2.
3.

## Checklist
